.github/workflows/nodejs.yml

```diff
@@ -18,4 +18,9 @@ jobs:
           cd src
           npm ci
           npm run build
-
+      - name: Run prettier
+        uses: creyD/prettier_action@v3.0
+        with:
+          dry: true
```

Comment:
How about we don't use dry mode? That way the action will automatically push a commit to the PR that formats the files, helping people that may not have Node & Prettier setup locally.

Reply:
It wont work.
For the action to push to PR back you need git password etc. . For how to do get that working with GA there is a example how to get it implemented on there site. See my comment about the example 2 and the internal source code how it work. 

There is also a option to amend the git. So no extra commit will be created with comment about prettier. Not everybody like this extra comment. Also not in the git squash commit text. But this is not important.

Reply:
It wouldn't use a password, it uses the `GITHUB_TOKEN` that is created by GitHub Actions automatically. I don't know whether it grants access to repo forks too though.
https://docs.github.com/en/free-pro-team@latest/actions/reference/authentication-in-a-workflow#permissions-for-the-github_token

Reply:
I'll give this a shot tomorrow.

Reply:
Creation of that "prettier" commit after a pull-request to master seems to be working on my fork.
Contributors with node/hooks set up on local machine will have no issues with this new GA (it's practically a no-op).
Contributors without one will get benefitted with auto code formatting greatly.
I think we can get best of both worlds here. Following @GerryFerdinandus's comment, I'm making those changes.

Reply:
Any updates @GerryFerdinandus?
I think we should go with failing on unformatted code, for now.
Maybe see if there are contributors who don't have nodejs setup.